src/store/settings/state.ts

```diff
@@ -38,6 +39,7 @@ function initialState(): SettingsState {
     runtimeVersion: runtimeVersion ? Number(JSON.parse(runtimeVersion)) : 0,
     runtimeVersionSubscription: null,
     networkFees: {} as NetworkFeesObject, // It won't be empty at the moment of usage
+    isDesktop: isElectron(), // NOTE: inverse flag here to debug desktop
```

Comment:
please put it into README

Reply:
Will we have separate README for wallet repo when it is migrated to DEX repo?

Reply:
have added